ci-operator/templates/openshift/openshift-azure/build-node-image.yaml

```diff
@@ -133,6 +136,13 @@ objects:
         sed -i -e "s#TENANT_ID#TENANT#" /tmp/shared/credentials
         sed -i -e "s#CLIENT_SECRET#SECRET#" /tmp/shared/credentials
         source /tmp/shared/credentials
+        
+        # after 3.10 decommission this will become default
```

Comment:
This is already the default in the template.

Reply:
What do you mean?

Reply:
`OPENSHIFT_RELEASE` defaults to 3.11 already

Reply:
I had in mind that "repo will become default", not version